src/main/java/core/basesyntax/Rectangle.java

```diff
@@ -0,0 +1,28 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Rectangle extends Figure {
+    public static final int MAXIMUM_SIZE = 50;
+    private int length;
+    private int width;
+    private Random random = new Random();
+
+    @Override
+    public void randomizeProperty() {
+        length = random.nextInt(MAXIMUM_SIZE);
+        width = random.nextInt(MAXIMUM_SIZE);
+    }
```

Comment:
The 'randomizeProperty' method should not be responsible for generating random values. Consider moving the random value generation to a separate class like 'FigureSupplier' and passing the values to the constructor of the figure.